lib/inherited_proofing/mhv/configuration.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module InheritedProofing
+  module MHV
+    class Configuration < Common::Client::Configuration::REST
+      def valid_id_url
+        Settings.mhv.inherited_proofing.valid_id_url
+      end
+
+      def vacct_info_url
+        Settings.mhv.inherited_proofing.vacct_info_url
+      end
+
+      def app_token
+        Settings.mhv.rx.app_token
```

Comment:
Let's add a line for `mhv.inherited_proofing.app_token`, I know it'll be the same as `rx.app_token` but would be good to copy it in case we need to do separate upkeep